infra/Makefile

```
@@ -0,0 +1,10 @@
+default: plan
+
+init:
+	cd envs/$(DEPLOYMENT_STAGE) && make init
```

Nit: I like this syntax more.
```suggestion
	make -C envs/$(DEPLOYMENT_STAGE) init
```

---

Leaving as is. Feel free to change on another PR if you have strong feelings